source/game/system/map/MapdataCheckPoint.hh

```diff
@@ -50,6 +50,10 @@ public:
     /// @endSetters
 
     /// @beginGetters
+    [[nodiscard]] u16 getEntryOffsetMs(const EGG::Vector2f &prevPos,
+            const EGG::Vector2f &pos) const;
+    [[nodiscard]] f32 getEntryOffsetExact(const EGG::Vector2f &prevPos,
+            const EGG::Vector2f &pos) const;
```

Comment:
These are non-trivial, should they really be in the "getter" section in the header?